### PR TITLE
New type definition for @ronomon/hash-table

### PR DESCRIPTION
**Pull request by airglow923**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/ronomon__hash-table/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
+// Type definitions for @ronomon/hash-table 2.0
+// Project: https://github.com/ronomon/hash-table#readme
+// Definitions by: airglow923 <https://github.com/me>
```
Comment (danger-public) [inline types/ronomon__hash-table/index.d.ts:3]: - :no_entry_sign: This line should have your github username in it, not /me

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## ronomon__hash-table ([<kbd>unpkg</kbd>](https://unpkg.com/browse/ronomon__hash-table@latest/))
was missing the following properties:
1. bucket
1. buckets
1. buffers
1. capacity
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 5414f9d0ff116d3952e372fd991c6b6a700e55ad
</p>

File: types/ronomon__hash-table/index.d.ts (hunk 1 above)
Comment (airglow923) [reply types/ronomon__hash-table/index.d.ts:3]: I do not own the `npm` package for `@ronomon/hash-table`, and this type definition is to support static type analysis for that package.

Comment (elibarzilay) [reply types/ronomon__hash-table/index.d.ts:3]: Nevertheless, there should be a human owner who will respond to reviews etc.  See [the description in the readme](https://github.com/DefinitelyTyped/DefinitelyTyped#definition-owners_ for what it is.

**Review by elibarzilay — CHANGES_REQUESTED** [review]

**Pull request merged**